### ISSUE TITLE
GeoMaps Language and Loading Error

### ISSUE THREAD

**Issue by tekduy** [wet-boew/wet-boew#8567]
When calling the geomaps function, the geomap loads in a language not native to my machine and proceeds to break the data table it is pulling from when selecting a page. Has anyone else had this issue and is there a fix for this?

![capture](https://user-images.githubusercontent.com/36016701/52654333-9de52600-2ea6-11e9-83db-8c17ab968981.PNG)
![capture2](https://user-images.githubusercontent.com/36016701/52654335-9de52600-2ea6-11e9-8cab-cf32936ad409.PNG)
![capture3](https://user-images.githubusercontent.com/36016701/52654336-9de52600-2ea6-11e9-8091-8af293cc3f39.PNG)

**Comment by duboisp** [comment]
What is the language of the page, the attribute ```lang``` set on the ```html``` element? What language are you expecting?

If the i18n string for geomap plugin are undefined, they fall back on English. It is possible to add new translation to our dictionary.

**Comment by tekduy** [comment]
There is no `lang` attribute set, so I would be expecting English. 

But I'm also coming across a second problem as well where geomaps is being called twice and as a result, bleeding into the data table and populating it with two selection options and two zoom options. However, this does not occur consistently.

![capture4](https://user-images.githubusercontent.com/36016701/52656250-c838e280-2eaa-11e9-81cb-d253b4f6b907.PNG)

**Comment by jvanulde** [comment]
@tekduy can you post your configuration and I can take a look.

**Comment by tekduy** [comment]
Here's how I'm calling geomaps:

`        <div id="mygeomap" class="wb-geomap position scaleline"
             data-wb-geomap='{
                    "tables": [{
                        "id": "stations",
                        "zoom": true,
                        "tab": false,
                        "visible": true,
                        "popups": true,
                        "style": {
                            "type": "symbol",
                            "init": {
                                "pointRadius": 4,
                                "strokeColor": "#FF0000",
                                "fillColor": "#FF0000",
                                "fillOpacity": 0.7
                            }
                        }
                    }]
                }'>`

`        <div class="row">
            <div class="col-md-9">
                <div class="wb-geomap-map"></div>
            </div>
            <div class="wb-geomap-legend col-md-3">
                <h2>Legend</h2>
            </div>
            <div class="row">
                <div class="wb-geomap-layers col-md-12"></div>
            </div>
        </div>
        </div>`

which links to my stations data table which is called with the class:
 `<table class="wb-tables table table-striped table-hover" table id="stations" aria-label="All Stations">`

**Comment by jvanulde** [comment]
@tekduy are you seeing the language issue with any other wet-boew plugin? I don't see any issue with your configuration.

**Comment by tekduy** [comment]
Yes, the datepicker plugin is also not loading in English.

![capture5](https://user-images.githubusercontent.com/36016701/52659730-e0146480-2eb2-11e9-8224-335373b21a1f.png)

**Comment by tekduy** [comment]
I've also tried excluding the `.js` files in `i18n` except for `en.js` and `en.min.js` from my project to force the map to load in English, but this did not work.

**Comment by jvanulde** [comment]
Strange. I would focus on solving the language issue first. Then we can tackle the Geomap issues.

**Comment by thekodester** [comment]
Just curious was the HTML tag set to the language? Like `<html class="no-js" lang="fr" dir="ltr">` is for French or for Chinese language you have `<html class="no-js" lang="zh" dir="ltr">`.

**Comment by RobJohnston** [comment]
He's stated that no `lang` attribute was set on the `html` tag.  So from https://www.w3.org/International/questions/qa-html-language-declarations

> For backwards compatibility, HTML5 describes an algorithm by which the default language of the content can be guessed at from the HTTP or meta Content-Language information under certain conditions. This is, however, only a fallback mechanism for cases where no language attribute has been used on the html tag. If you have used the language attribute on the html tag, as you always should, such fallbacks are irrelevant.

My guess is that the fallback algorithm is choosing Chinese.

**Comment by jvanulde** [comment]
Good point @RobJohnston, @tekduy please set the language explicitly and let us know if that solves the issue.

**Comment by tekduy** [comment]
When the language is set explicitly, this fixes the language issue but the loading of the geomap is still an issue.

**Comment by tekduy** [comment]
Perhaps it's how I'm bundling wet-boew? I'm on Windows 10 and using ASP.NET MVC 5. 
I'm bundling `theme.css` and `theme.js`  in `BundleConfig.cs` through:

`bundles.Add(new StyleBundle("~/Content/Render/canada_theme").IncludeDirectory(
                "~/Content/canada_theme", "*.css", true));`

`bundles.Add(new ScriptBundle("~/bundles/wet-boew").IncludeDirectory(
                "~/Scripts/wet-boew/", "*.js", false));`

Then calling them in `_Layout.cshtml` through:

`<head>`
`@Styles.Render("~/Content/Render/canada_theme")` 
`</head>`

`<body>`
`...`
`...`
`@Scripts.Render("~/Scripts/wet-boew/js/wet-boew.js")`
`@Scripts.Render("~/bundles/wet-boew")`
`<body>`

The contents of my `Contents` and `Scripts` folders are as such:

![capture6](https://user-images.githubusercontent.com/36016701/52862498-446d3900-30ea-11e9-92c2-edc54c319fc9.PNG)
![capture7](https://user-images.githubusercontent.com/36016701/52862508-4636fc80-30ea-11e9-9330-ed87ee72bbd2.PNG)

**Comment by RobJohnston** [comment]
Maybe you are bundling both the full an minimized versions with this?
```c#
bundles.Add(new ScriptBundle("~/bundles/wet-boew").IncludeDirectory( "~/Scripts/wet-boew/", "*.js", false))
```

Wouldn't it be better to change `*.js` to `*.min.js`?  That's my guess.
